simulators/ethereum/engine/helper/helper.go

```diff
@@ -53,7 +54,10 @@ func (rt *LoggingRoundTrip) RoundTrip(req *http.Request) (*http.Response, error)
 		return nil, err
 	}
 	reqLogBytes := bytes.TrimSpace(reqBytes[:])
-	if len(reqLogBytes) > MAX_LOG_BYTES {
+
+	hiveLogLevel, _ := strconv.Atoi(os.Getenv("HIVE_LOGLEVEL"))
```

Comment:
I think we could do this when instantiating the `LoggingRoundTrip` to not have to parse this from the env variable on every single HTTP call we do. Seems expensive.

Reply:
Ahh ofc! Sheesh - will change it :D

Reply:
Updated this in: https://github.com/ethereum/hive/pull/1066/commits/c08b4bbf39e7df8ae73e4e5135e8541f9c6493bb